### PR TITLE
Feature #388

### PR DESCRIPTION
**Pull request by Simple-codinger**

### CODE REVIEW

File: Integreat/Integreat.Shared/ViewModels/ExtrasContentPageViewModel.cs [hunk 1]
```diff
@@ -39,6 +41,19 @@ public class ExtrasContentPageViewModel : BaseContentViewModel
...
             Extras = new ObservableCollection<ExtraAppEntry>();
+
+            ChangeLanguageCommand = new Command(OnChangeLanguage);
+
+            // add toolbar items
+            ToolbarItems = new List<ToolbarItem>
+            {
+                new ToolbarItem { Text = AppResources.Language, Icon = "translate", Order = ToolbarItemOrder.Primary, Command = ChangeLanguageCommand },
+#if __ANDROID__
+                new ToolbarItem { Text = AppResources.Share, Order = ToolbarItemOrder.Secondary, Icon = "share", Command = ContentContainerViewModel.Current.ShareCommand },
```
Comment (thimmy687) [inline Integreat/Integreat.Shared/ViewModels/ExtrasContentPageViewModel.cs]: We can move this to separate methods

File: Integreat/Integreat.Shared/ViewModels/ExtrasContentPageViewModel.cs [hunk 2]
```diff
@@ -65,6 +86,14 @@ private void InvokeOnTap(object obj)
             extraAppEntry?.OnTapCommand?.Execute(obj);
         }
 
+        private async void OnChangeLanguage(object obj)
+        {
+            if (IsBusy) return;
+
+            ContentContainerViewModel.Current.OpenLanguageSelection();
+            return;
```
Comment (thimmy687) [inline Integreat/Integreat.Shared/ViewModels/ExtrasContentPageViewModel.cs]: Remove return

File: Integreat/Integreat.Shared/ViewModels/MainContentPageViewModel.cs [hunk 3]
```diff
@@ -75,14 +74,20 @@ public class MainContentPageViewModel : BaseContentViewModel
...
-            // add search icon to toolbar
+            // add toolbar items
             ToolbarItems = new List<ToolbarItem>
             {
-                new ToolbarItem { Text = AppResources.Search, Icon = "search", Command = OpenSearchCommand},
+                new ToolbarItem { Text = AppResources.Search, Icon = "search", Order = ToolbarItemOrder.Primary, Command = OpenSearchCommand},
+                new ToolbarItem { Text = AppResources.Language, Icon = "translate", Order = ToolbarItemOrder.Primary, Command = ChangeLanguageCommand },
+#if __ANDROID__
+                new ToolbarItem { Text = AppResources.Share, Order = ToolbarItemOrder.Secondary, Icon = "share", Command = ContentContainerViewModel.Current.ShareCommand },
+                new ToolbarItem { Text = AppResources.Location, Order = ToolbarItemOrder.Secondary, Command = ContentContainerViewModel.Current.OpenLocationSelectionCommand },
```
Comment (thimmy687) [inline Integreat/Integreat.Shared/ViewModels/MainContentPageViewModel.cs]: Duplicate in ExtrasContentPageViewModel

Comment (Simple-codinger) [reply Integreat/Integreat.Shared/ViewModels/MainContentPageViewModel.cs]: Yes there are duplications in all the BaseContentVieModels but it's the only way to get is work in iOS

**Pull request merged**